build.gradle

```diff
@@ -0,0 +1,44 @@
+plugins {
+    id 'java'
+    id 'maven-publish'
+}
+
+group 'com.google.cloud.dataproc.jdbc'
```

Comment:
`com.google.cloud.dataproc.jdbc` group seems to be too specific - should it be `com.google.cloud.dataproc` group with `dataproc-jdbc-connector` artifact?

Reply:
Changed, though I thought artifact corresponds to the class name?